core/common/src/main/java/alluxio/metrics/MetricsSystem.java

```diff
@@ -56,7 +56,8 @@
   public enum InstanceType {
     MASTER("master"),
     WORKER("worker"),
-    CLIENT("client");
+    CLIENT("client"),
+    CLUSTER("cluster");
```

Comment:
Does this mean we can get rid of the String constant?

Reply:
We used the constant for string comparison without transformation.

Reply:
Could we not use `InstanceType.CLUSTER.toString()`?